internal/cli/iam/projects/teams/add.go

```diff
@@ -79,7 +79,7 @@ func AddBuilder() *cobra.Command {
 
 	cmd.Flags().StringSliceVar(&opts.roles, flag.Role, []string{}, usage.TeamRole)
 
-	cmd.Flags().StringVar(&opts.OrgID, flag.ProjectID, "", usage.ProjectID)
+	cmd.Flags().StringVar(&opts.ProjectID, flag.ProjectID, "", usage.ProjectID)
```

Comment:
the importance of testing 😬

Reply:
:P 
I actually have a PR in the works to start testing what flags we set to a command